app/models/goal.py

```diff
@@ -1,5 +1,23 @@
-from sqlalchemy.orm import Mapped, mapped_column
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+
 from ..db import db
 
+
 class Goal(db.Model):
     id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
+    title: Mapped[str]
+    tasks = relationship('Task', back_populates='goal', cascade='all, delete-orphan')
+
+    def to_dict(self, noTasks=True):
```

Comment:
we should use snake case for variables in python

```suggestion
    def to_dict(self, no_tasks=True):
```